### ISSUE TITLE
Modifying Apache port result in wrong requests path

### ISSUE THREAD

**Issue by chipironcin** [bitnami/bitnami-docker-phabricator#45]
Hi guys,
I want to deploy Phabricator in my headless server, but I want to access it from my computer locally (`127.0.0.1:<port>`).
I have an ssh tunnel from my computer to my server, that actually links the port 80 on my server with the port 10080 of my computer.

The setup worked by default. But as Phabricator base-uri is `127.0.0.1` by default, it expects me to connect with that url, but I am reaching it like `127.0.0.1:10080`. It refuses to work.

Ok, lets make Apache and Phabricator listen on 10080 inside the container, then docker to link 100080 from the container to 80 locally in my server. Then tunneling my server 80 port as 10080 locally in my machine.

First I went to the docker-compose.yml and defined the the Apache Http Port under the `phabricator` service like
```
    environment:
      - APACHE_HTTP_PORT=10080
```

Then I also modified the `phabricator` section to make the ports match like
```
    ports:
      - '80:10080'
      - '443:443'
```
--------------------

With this setup, Apache should be listening on 10080 inside the container and Phabricator configured with that port. Then Docker should be linking 10080 to 80. Then with the tunnel I am making `server:80` look like `127.0.0.1:10080` in my machine.

It works but assets are being loaded without the port and I cannot login after clicking on login ( requests are going to `127.0.0.1` instead of `127.0.0.1:10080`) so I cannot fully work with Phabricator.
When I log into the container and execute `bin/config get phabricator.base-uri` it returns `127.0.0.1` instead of `127.0.0.1:10080`.

My guess is that the issue is there. Can anyone confirm and try to fix it? Thanks!

**Comment by chipironcin** [comment]
I can confirm executing manually `bin/config set phabricator.base-uri 'http://127.0.0.1:10080'` made it work again. Phabricator configuration seems to not be reading the server http port well.

**Comment by juan131** [comment]
@chipironcin we have identified the problem. This issue will be fix in the new release coming.

**Comment by juan131** [comment]
Issue fixed on new release 2017.04-​r0

**Comment by juan131** [comment]
@chipironcin could you confirm us that the issue is solved on the new release?

**Comment by chipironcin** [comment]
I confirm this has been fixed on the latest release (2017.04-​r0)
Awesome! Thanks for the fix and keep up the good work!
